<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="72918" started-at="2017-03-31T02:26:48Z" finished-at="2017-03-31T02:28:01Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="72918" started-at="2017-03-31T02:26:48Z" finished-at="2017-03-31T02:28:01Z">
      <class name="testCases.NewTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:testCases.NewTest@28f67ac7]" name="beforeTest" is-config="true" duration-ms="211" started-at="2017-03-30T19:26:48Z" finished-at="2017-03-30T19:26:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testCases.NewTest@28f67ac7]" name="beforeMethod" is-config="true" duration-ms="15477" started-at="2017-03-30T19:26:48Z" finished-at="2017-03-30T19:27:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="loginTest()[pri:0, instance:testCases.NewTest@28f67ac7]" name="loginTest" duration-ms="27045" started-at="2017-03-30T19:27:04Z" finished-at="2017-03-30T19:27:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:testCases.NewTest@28f67ac7]" name="afterMethod" is-config="true" duration-ms="4464" started-at="2017-03-30T19:27:31Z" finished-at="2017-03-30T19:27:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginTest status=SUCCESS method=NewTest.loginTest()[pri:0, instance:testCases.NewTest@28f67ac7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testCases.NewTest@28f67ac7]" name="beforeMethod" is-config="true" duration-ms="9947" started-at="2017-03-30T19:27:35Z" finished-at="2017-03-30T19:27:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="registrationTest()[pri:0, instance:testCases.NewTest@28f67ac7]" name="registrationTest" duration-ms="10" started-at="2017-03-30T19:27:45Z" finished-at="2017-03-30T19:27:45Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at testCases.NewTest.registrationTest(NewTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:648)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:834)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationTest -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:testCases.NewTest@28f67ac7]" name="afterMethod" is-config="true" duration-ms="2713" started-at="2017-03-30T19:27:45Z" finished-at="2017-03-30T19:27:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=registrationTest status=FAILURE method=NewTest.registrationTest()[pri:0, instance:testCases.NewTest@28f67ac7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testCases.NewTest@28f67ac7]" name="beforeMethod" is-config="true" duration-ms="9266" started-at="2017-03-30T19:27:48Z" finished-at="2017-03-30T19:27:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="xyzTest()[pri:0, instance:testCases.NewTest@28f67ac7]" name="xyzTest" duration-ms="1" started-at="2017-03-30T19:27:57Z" finished-at="2017-03-30T19:27:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- xyzTest -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:testCases.NewTest@28f67ac7]" name="afterMethod" is-config="true" duration-ms="3338" started-at="2017-03-30T19:27:57Z" finished-at="2017-03-30T19:28:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=xyzTest status=SUCCESS method=NewTest.xyzTest()[pri:0, instance:testCases.NewTest@28f67ac7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:testCases.NewTest@28f67ac7]" name="endReport" is-config="true" duration-ms="423" started-at="2017-03-30T19:28:01Z" finished-at="2017-03-30T19:28:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- testCases.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
